Configuration/StandardSequences/python/SimL1EmulatorRepack_FullSimTP_cff.py

```diff
@@ -114,7 +140,7 @@
 
 
 SimL1EmulatorTask = cms.Task()
-stage2L1Trigger.toReplaceWith(SimL1EmulatorTask, cms.Task(unpackEcal,unpackHcal,unpackCSC,unpackDT,unpackRPC,unpackEmtf,unpackCsctf,unpackBmtf
+stage2L1Trigger.toReplaceWith(SimL1EmulatorTask, cms.Task(unpackEcal,unpackHcal,unpackCSC,unpackDT,unpackRPC,unpackGEM,unpackEmtf,unpackBmtf,unpackOmtf
```

Comment:
Small point: iiuc, GEM has its own modifier, so maybe something like the following should be done.
https://github.com/cms-sw/cmssw/blob/1e7c61e8e118871e99e5aa53bd6b396101f5eb75/Configuration/StandardSequences/python/SimL1EmulatorRepack_Full_cff.py#L157-L165